### PR TITLE
Animated Text: Jungle Inferno

### PR DESCRIPTION
**Pull request by LilyMakesThings**
- Added 13 new blocks that are not compatible with Scratch Lab
- Added button to enable incompatible features:
    - Shows the TurboWarp-only blocks
    - ~~Adds new fonts to the "set font" menu~~ edit from maintainers: removed for now, plans to add back later with cross platform support

image

### CODE REVIEW

**Comment by horizontal-shading** [comment]
Cool

**Comment by SurvExe1Pc** [comment]
i like this

**Comment by LilyMakesThings** [comment]
oh I forgot to finish programming something lmao, gimme a sec

**Comment by LilyMakesThings** [comment]
I want to reconsider some of the blocks in this and potentially replace them with some more reasonable ones

(it should not take 2 to 3 blocks to have a rainbow effect that lasts a certain amount of seconds)

**Comment by echoless3484** [comment]
Nice, though there's a text width and align block combined already.

**Comment by BlueDome77** [comment]
Could you add this useful block?

Screenshot 2023-07-13 at 8 27 21 PM

**Comment by LilyMakesThings** [comment]
> Nice, though there's a text width and align block combined already.

The point was I wanted to divorce them

**Comment by LilyMakesThings** [comment]
> Could you add this useful block?
> 
> Screenshot 2023-07-13 at 8 27 21 PM

This block was added in the last commit

**Comment by echoless3484** [comment]
I had an idea, is it possible to make the extension fetch a list of compatible fonts, so people can simply add to the list? is does it need the files with it

**Comment by 0832k12** [comment]
You do that I want to do.

**Comment by David-Orangemoon** [comment]
Oh no its the last update for 1486 days.

**Comment by LilyMakesThings** [comment]
> @LilyMakesThings Your branch is out-of-date with the base branch

I'll get it to it one a day that doesn't end in Y

**Comment by horizontal-shading** [comment]
This extension is coming out February 30th

File: extensions/lab/text.js [hunk 1]
```diff
@@ -29,6 +31,22 @@
...
+    'Scratch',
+    'Arial',
+    'Arial Black',
+    'Helvetica Neue',
+    'Calibri',
+    'Garamond',
+    'Times New Roman',
+    'Courier New',
+    'Brush Script MT',
+    'Impact',
```
Comment (GarboMuffin) [inline extensions/lab/text.js:44]: a lot of these are not going to work on Chrome OS (~32% of users), macOS (~7% of users), or Linux (~5% of users)

Comment (GarboMuffin) [reply extensions/lab/text.js:44]: fonts have always been tricky for this reason

Comment (LilyMakesThings) [reply extensions/lab/text.js:44]: Should they be removed?

Comment (GarboMuffin) [reply extensions/lab/text.js]: removing them for now

**Pull request merged**